docs/source/developer/contributing.rst

```diff
@@ -38,6 +28,20 @@ developers to keep up with.
 Note that the Github issue tracker is the best place to report bugs.  Bug
 reports in chat are difficult to track and likely to be lost.
 
+Forum
+.....
+
+Numba uses Discourse as a forum for longer running threads such as design
+discussions and roadmap planning. There are various categories available and it
+can reached at: `numba.discourse.group https://numba.discourse.group/`_.
```

Comment:
```suggestion
can reached at: `numba.discourse.group <https://numba.discourse.group/>`_.
```

Reply:
I fixed the link already. However, additionally there is a 'be'  missing here.

Reply:
build is still failing, I'm pretty sure this syntax is needed?!

Reply:
Indeed.